### PR TITLE
Training doc update

### PR DESCRIPTION
**Pull request by henry88lay**
Here are all the updates and changes I did to the training as I went. I Will be continuing my updates for the remaining pages on gitbooks

### CODE REVIEW

File: hands-on-challenges/2.-using-source-format.md [hunk 1]
```diff
@@ -49,18 +49,18 @@ ERROR running force:mdapi:convert:  Unexpected file found in package directory:
 
 Observe the difference between mdapi and source format. Object Metadata is decomposed into multiple files, this helps when you have a large project with multiple objects having a large amount of fields. Tracking the changes in fields and record types is relatively easy and time saving. 
 
-All challenges from here on will use source format 
+All challenges from here on will use **source format** 
 
 {% hint style="warning" %}
 When running `sfdx force:mdapi:convert` LWC bundle is missing .js-meta.xml file. This is sfdx CLI bug [https://github.com/forcedotcom/cli/issues/723](https://github.com/forcedotcom/cli/issues/723)  you can copy .js-meta.xml file from [https://github.com/dxatscale/dxatscale101/tree/master/mdapi/lwc](https://github.com/dxatscale/dxatscale101/tree/master/mdapi/lwc)
```
Comment (azlam-abdulsalam) [inline hands-on-challenges/2.-using-source-format.md:55]: @henry88lay  is this bug still exists?

Comment (henry88lay) [reply hands-on-challenges/2.-using-source-format.md:55]: from my testing hasn't poped up at all

**Review by azlam-abdulsalam — CHANGES_REQUESTED** [review]
Check the comments

File: hands-on-challenges/2.-using-source-format.md (hunk 1 above)
Comment (azlam-abdulsalam) [reply hands-on-challenges/2.-using-source-format.md:55]: so can we clean this up then?

**Pull request merged**